static/js/utils.js

```diff
@@ -1,5 +1,8 @@
 function alert(type, message, delay) {
   console.info(type, message)
+  if(!message) {
+    message = type
+  }
   let notification = document.querySelector('#notification')
```

Comment:
`const notification = document.querySelector('#notification') || document.createElement('div')`
But why would `#notification` not be found? Isn't it supposed to be in the layout?
Notification is not just a plain blank `div`, it has attributes.

Reply:
you write this ... :)

Reply:
Well, you let this go through the review without giving an explicit reason? 😛 
It's either not applicable anymore, or it's missing a comment to explain why I needed to do this :/

Reply:
I will cut and paste your comments in the new PR